oeps/processes/oep-0055-proc-project-maintainers.rst

```diff
@@ -0,0 +1,428 @@
+owned both in the repository and in production; owners answered the
+pager when something went wrong.
+
+What Maintainers Do
+-------------------
+The role of the repository or project maintainer is both a community
+focused and a technical role [#role]_.  Maintainers will be the public face
+of each component and should be welcoming and available.  While
+maintainers should strive to always be friendly and kind in their
+communication, firmness and directness are equally important.
+Maintainers must enforce the project's standards.  In cases where a
+proposed change isn't aligned with the our standards or direction, it
+is best to make that clear as early as possible.
+
+Non-technical responsibilities of maintainers include:
+
+* Effective communication with community members, other maintainers,
+  and project leadership
+* Selection and approval of Core Contributors with commit rights to their repositories
+* Disciplined management to ensure that maintenance work is done
+* Enforcement of the project's standards, both technical and social: maintainers
+  must remain friendly, kind, and open, but that should not supercede
+  enforcement of technical standards
+* The ability to solve or escalate issues facing users of the
+  component
+
+Technical responsibilities of maintainers include:
+
+* Understanding the current component architecture and implementation
+* Ensuring that basic documenation for the component exists and is
+  accurate
+* Having a documented technical vision for the evolution of the
+  component
+* The ability to assess the appropriateness of proposed changes to or
+  extensions of the component
+* The role of the component in the overall platform and how it
+  interacts with other components
+* Must be available to participate in the comment period for Core Contributor
+  nominees nominated to commit code to the maintainer's repo(s), as part of the
+  `Core Contributor Program`_
+  
+It is acceptable for a component to be maintained by a team or group
+as long as that arrangement does not reduce the reachability of at least one
+maintainer or reduce accountability for the above responsibilites. The
+maintaining team must have ways of direct contact, such as an email address and
+a GitHub team, that anyone can use to get ahold of someone - it is completely
+acceptable for the team to figure out arrangements for list monitoring, such as
+an "on-call" rotation.
+
+Prerequisites
+-------------
+In order to be a repository maintainer, you must
+
+* Be an active member of the Open edX community who has made technical
+  contributions to the project in the last 6 months.  The types of
+  contributions that are relevant will match those defined in the
+  `code contributors`_
+* Abide by the Open edX `Code of Conduct`_
+  <https://openedx.org/code-of-conduct/>`_
+* Be a member of the `Core Contributor Program`_ with rights to the repository
+  in question who has demonstrated excellence in the role of core contributor.
```

Comment:
First off, legally, to get write access to a repo, one will need to formally be in the Core Contributor program.

Secondly, I've been reading about communities. I'm not very comfortable having a program that feels so exclusive, or a program that has such strict standards that people lose status easily. This is a topic I would like to discuss further and I think in person would be good. See https://discuss.openedx.org/t/conf-bof-governance-topics/6841 and in there a reference I quote.

Reply:
OK, I definitely hear you and agree with many of your sentiments - I look forward to discussing more at the conference.

My concern (as I also raised in a discourse thread about granting write access to WG repos) is proliferating different legal agreements and expecting someone (particularly, myself and the legal team at tCRIL) to monitor everyone's activities and what they're doing with their permissions, and have them sign new agreements when they move into different categories, possibly having to invalidate old agreements... it becomes a headache to administer. So - perhaps selfish, but also (given the burden on our legal counsel already) realistic.

Reply:
@antoviaque do you think landing this conversation is blocking for merging this?  I do agree when need to find and strike the right balance between opportunities to participate and accountability to contribute.  This feels like a great conversation to have at the conference as part of the Friday session on shared ownership.

Reply:
@e0d I think it would be useful to clarify it yes -- the core contributor program is still quite young, so it will be important to have clear definitions of both programs and status, so we don't end up with confusion about it. Imho it needs to be clear what the incentive for organizations to keep their core contributor commitment is, vs maintainership -- especially if one of the two roles have a lower threshold.

Discussing it at the conference sounds good! I'll suggest the topic in the thread @sarina started for collecting governance topics to discuss (https://discuss.openedx.org/t/conf-bof-governance-topics/6841)

Reply:
@antoviaque and @sarina I'm going to attempt to summarize our conversation.

We agreed that we're going to keep the requirement that one must be a Core Contributor to become a Maintainer for now.  If we find that that limits participation in practice -- not a goal -- we'll revisit.

We value the level of commitment that comes with being a Core Contributor and don't want to reduce that.  We want to ensure that the programs are congruent and not working at cross purposes.  Hours spent maintaining components would obviously count toward a Core Contributors commitment.  

If we see issues that need to be addressed we can propose amendments to this OEP as we learn.

Reply:
@e0d - yes, that is what I took away from our conversation. Let's see how this works, and adjust if necessary.